akka-docs/rst/scala/dispatchers.rst

```diff
@@ -40,9 +40,9 @@ is to configure the dispatcher:
   threads the pool keep running in order to reduce the latency of handling a new incoming task.
   You can read more about parallelism in the JDK's `ForkJoinPool documentation`_.
 
-And here's another example that uses the "thread-pool-executor":
-
-.. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#my-thread-pool-dispatcher-config
+another example that uses the "thread-pool-executor":
+ 
+ .. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#fixed-pool-size-dispatcher-config
```

Comment:
Intended indent of one space, or is it the other ones that are missing the indent?